README.md

```diff
@@ -21,7 +21,7 @@
 In order to build this project, you need to first clone the repo and then run `make install`:
 
 ```sh
-> git clone https://github.com/ipfs/http-api-docs/http-api-docs
+> git clone https://github.com/ipfs/http-api-docs
```

Comment:
This change conflicts with #15, which I *think* is the right way to go with this for now. Do you mind reverting your change here so that it merges cleanly with that PR? Thanks!